### PR TITLE
Conditional foundation import

### PR DESCRIPTION
**Pull request by thomasvl**

### CODE REVIEW

**Comment by thomasvl** [comment]
@gjcairo fyi

**Review by allevato — APPROVED** [review]

File: Sources/protoc-gen-swift/Descriptor+Extensions.swift [hunk 1]
```diff
@@ -15,6 +15,22 @@ extension FileDescriptor {
     return SwiftProtobufInfo.isBundledProto(file: self)
   }
 
+  // Returns true if the file will beed to import Foundation.
```
Comment (gjcairo) [inline Sources/protoc-gen-swift/Descriptor+Extensions.swift:18]:
```suggestion
  // Returns true if the file will need to import Foundation.
```

File: Sources/protoc-gen-swift/Descriptor+Extensions.swift [hunk 2]
```diff
@@ -166,6 +182,22 @@ extension Descriptor {
   /// Returns true if the message should use the message set wireformat.
   var useMessageSetWireFormat: Bool { return options.messageSetWireFormat }
 
+  var needsFoundationImport: Bool {
```
Comment (gjcairo) [inline Sources/protoc-gen-swift/Descriptor+Extensions.swift:185]: Nit: can we add the same docs from FileDescriptor above to this?

**Review by gjcairo — COMMENTED** [review]
LGTM but two doc nits.

Thanks Thomas!

**Comment by thomasvl** [comment]
Oops, window didn't reload to show the comment until after I hit merge.  Follow up PR coming.

**Pull request merged**